publish/publish.proj

```diff
@@ -11,9 +11,13 @@
   
   <Target Name="PublishFinalOutput"
           Condition="'$(Finalize)' == 'true'"
-          DependsOnTargets="PublishCoreHostPackagesToFeed;FinalizeBuildInAzure;UpdatePublishedVersions" />
-          
-  <Target Name="ExcludeSymbolsPackagesFromPublishedVersions" BeforeTargets="UpdatePublishedVersions" >
+          DependsOnTargets="PublishCoreHostPackagesToFeed;FinalizeBuildInAzure;UpdateVersionsRepo" />
+
+  <Target Name="UpdateVersionsRepo"
+          Condition="'$(GitHubAuthToken)' != ''"
```

Comment:
Right now, I just copied what I did for 1.0.0/1.1.0.

I think it makes sense to skip based on auth key/token only and error on other blanks, because in some cases (e.g. dotnet/versions repo name) there are defaults--and api keys never have defaults. Erroring out rather than skipping when the user missed some non-obvious config sounds easier to figure out to me.